scripts/tests/chiptest/test_definition.py

```diff
@@ -238,7 +233,9 @@ def Run(self, runner, apps_register, paths: ApplicationPaths, pics_file: str):
             # Remove server application storage (factory reset),
             # so it will be commissionable again.
             app.factoryReset()
-            app.start(str(randrange(1, 4096)))
+            # Create dictionary for command line options for starting the App
+            commandLineOptions = {"--discriminator": str(randrange(1, 4096))}
```

Comment:
I wonder if we can just remove this line. It has been added in the past because of the cross-talks between darwin instances. But I don't think it happens anymore.

This is good to keep the option to configure the discriminator from the script the default random parameter is probably useless now. 
@bzbarsky-apple You added it iirc, any reason to not just remove it now ? Personally I like the randomness, but thinking about it we can also create a YAML script that test some values and the value limits in a deterministic way.
And this can probably be a separate PR to add this YAML file that test the different values.

Reply:
I don't think there's any reason not to remove it, except for the "it's nice to test that things still work" aspect....

Reply:
I've removed it and will let the default discriminator kick in during app launch